Makefile

```diff
@@ -338,6 +338,7 @@ define build-and-tag # (service, image, mutable_image, prefix)
 	# -i.bak is required for cross-platform compat: https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux
 	sed -i.bak "s|BASEIMAGE|$(BASEIMAGE)|g" $(tmp_build_path)/Dockerfile
 	rm $(tmp_build_path)/Dockerfile.bak
+	[ "$(ARCH)" != "amd64" ] && docker run --rm --privileged multiarch/qemu-user-static:register --reset
```

Comment:
Looked through https://hub.docker.com/r/multiarch/qemu-user-static/ and the associated github entry. Did not dig much deeper to understand how it works. Seems neat. 

Do we need the conditional? Seems to be supported on other archs already.

Reply:
I was following the strategy used in https://github.com/kubernetes/kubernetes/blob/a87057bc6f46c294e0ba4cc28059415129f729a4/build/debian-base/Makefile#L61. It's not harmful to register the qemu statics even when the target architecture is amd64 (assuming that the machine that you are building with supports it). The conditional as is is returning a non-zero error when the target architecture is amd64, so I'll just remove it rather than figure out the right syntax for an if statement in sh (and whatever other shells happen to be used).